### PR TITLE
Show "upload percentage" at the center of the screen. 

### LINKED ISSUE #1590: Hope to let filebrowser display the transfer rate when uploading.
My upload rate is not particularly fast, and not seeing the upload rate makes me a little anxious.
Show the rate when the mouse hovers over the upload progress bar, or show the speed when the mouse hovers over the upload button, which is great.

### PR DESCRIPTION
**Pull request by niubility000**
fixed #1590

### CODE REVIEW

**Comment by niubility000** [comment]
Yes, it may look a little bit ugly. Who can make it better?  I am not good at it.

File: frontend/src/views/Layout.vue [hunk 1]
```diff
@@ -9,7 +9,15 @@
       <shell v-if="isExecEnabled && isLogged && user.perm.execute" />
     </main>
     <prompts></prompts>
+
+    <div v-if="this.progress !==0" >
+       <div style=" height: 8rem;  width: 100%;  z-index: 99999;  position: fixed; top:40%; text-align:center; font-size:6em; "> {{ this.progress }}%</div>
```
Comment (mckaygerhard) [inline frontend/src/views/Layout.vue:14]: i think is not so ugly.. the only hint is move the style to the css at `frontend/src/css/styles.css` and that's all

**Comment by daiaji** [comment]
The percentage of uploads that are always placed in the center of the screen may distract users from doing other things.

**Comment by mckaygerhard** [comment]
> The percentage of uploads that are always placed in the center of the screen may distract users from doing other things.

ok but i guess that know if upload are doing well is more important.. of course the percent can be moved away to an area of notification but will need more coding and this merge request is a collaboration.. 

so then you can propose some changes later

**Comment by o1egl** [comment]
> > The percentage of uploads that are always placed in the center of the screen may distract users from doing other things.
> 
> ok but i guess that know if upload are doing well is more important.. of course the percent can be moved away to an area of notification but will need more coding and this merge request is a collaboration..
> 
> so then you can propose some changes later

Merging this PR will block the release process until we find a better UX solution for it. This is not acceptable

**Pull request closed without merging**